sklearn/datasets/lfw.py

```diff
@@ -369,7 +368,7 @@ def _fetch_lfw_pairs(index_file_path, data_folder_path, slice_=None,
     # parse the index file to find the number of pairs to be able to allocate
     # the right amount of memory before starting to decode the jpeg files
     with open(index_file_path, 'rb') as index_file:
-        split_lines = [ln.strip().split(b('\t')) for ln in index_file]
+        split_lines = [ln.decode().strip().split('\t') for ln in index_file]
```

Comment:
Wouldn't this make `split_lines` a list of `str` instead of `bytes` though?

Reply:
yes. Is that an issue?